applications/CoSimulationApplication/python_scripts/development_helpers/dummy_solver_wrapper.py

```diff
@@ -0,0 +1,43 @@
+from __future__ import print_function, absolute_import, division  # makes KratosMultiphysics backward compatible with python 2.6 and 2.7
+
+# Importing the Kratos Library
+import KratosMultiphysics as KM
+
+# Importing the base class
+from KratosMultiphysics.CoSimulationApplication.base_classes.co_simulation_solver_wrapper import CoSimulationSolverWrapper
+
+# Other imports
+import KratosMultiphysics.CoSimulationApplication.co_simulation_tools as cs_tools
+
+def Create(settings, solver_name):
+    return DummySolverWrapper(settings, solver_name)
+
+class DummySolverWrapper(CoSimulationSolverWrapper):
+    """This class serves as dummy for testing, it does not solve anything
+    It only imports a mesh that can be used in the testing workflow
+    """
+    def __init__(self, settings, solver_name):
+        super(DummySolverWrapper, self).__init__(settings, solver_name)
+
+        self.time_step = self.settings["solver_wrapper_settings"]["time_step"].GetDouble()
+        self.model_part = self.model.CreateModelPart(self.settings["solver_wrapper_settings"]["main_model_part_name"].GetString())
+
+        self.model_part.ProcessInfo[KM.DOMAIN_SIZE] = self.settings["solver_wrapper_settings"]["domain_size"].GetInt()
+
+        cs_tools.AllocateHistoricalVariablesFromCouplingData(self.data_dict.values(), self.model, self.name)
+
+    def Initialize(self):
+        severity = KM.Logger.GetDefaultOutput().GetSeverity()
+        KM.Logger.GetDefaultOutput().SetSeverity(KM.Logger.Severity.WARNING) # mute MP-IO
+        model_part_io = KM.ModelPartIO(self.settings["solver_wrapper_settings"]["mdpa_file_name"].GetString())
+        model_part_io.ReadModelPart(self.model_part)
+        KM.Logger.GetDefaultOutput().SetSeverity(severity)
```

Comment:
Does this work with pyKratos ?

Reply:
nope, but neither does the entire file
It is only an example